docs/stdlib/generic.rst

```diff
@@ -134,11 +146,11 @@ Generic
 
 .. eql:operator:: lt: anytype < anytype -> bool
 
-    Less than operator.
+    Performs a less than value comparison with two valeues of
```

Comment:
```suggestion
    Produces true for two compatible values if the left is the smaller value.
```

Would like to style `true` here as syntax, but I think not doing so is a reasonable compromise to have an definition that fits within our space constraints. Open to other suggestions.